### DISCUSSION TITLE
Grant Program

### DISCUSSION THREAD

**Discussion by 0xmatt1** [kcc-community/any-advice-issue#14] [Q&A]
Hi, interested in contributing, is there a grant program available?

**Comment by viaweb3** [comment]
what is a grant program?

**Reply by BarneyChambers** [reply]
I think he is referring to a pool of funds (maybe KCS?) that is allocated to community developers for building good stuff on the platform

**Reply by caizhiyuanrise** [reply]
> what is a grant program?

A grant is a Fund given to developers with no return asked ever. 
You submit your project you want to establish and the amount you need to make this happen, and then the team decides if they want to find your project. 

If you search EPIC Megagrant. Then you can see the same for Game-Developers in the UnReal universe. You can get 25K and no one ever asks for any return. If you succeed or not.

**Comment by viaweb3** [comment]
yes, we are planning it.

**Comment by 0xmatt1** [comment]
I would reference the grant program web3, consensus, and polkadot uses as a
good framework / example of a fairly decent functioning program , people
don't get money and get to walk away. There's a process its all done in to
keep all parties mutually involved
